File: MifosXDroid/src/main/java/com/mifos/objects/Page.java
package com.mifos.objects;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by ishankhanna on 09/02/14.
 */
public class Page {

    private int totalFilteredRecords;
    private List<PageItem> pageItems = new ArrayList<PageItem>();


    public List<PageItem> getPageItems() {
        return pageItems;
    }
    public void setPageItems(List<PageItem> pageItems) {
        this.pageItems = pageItems;
    }


    public int getTotalFilteredRecords() {
        return totalFilteredRecords;
    }

    public void setTotalFilteredRecords(int totalFilteredRecords) {
        this.totalFilteredRecords = totalFilteredRecords;
    }


}

File: core/common/src/main/java/com/mifos/core/common/utils/Page.kt
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.common.utils

/**
 * Created by ishankhanna on 09/02/14.
 */
data class Page<T>(
    var totalFilteredRecords: Int = 0,

    var pageItems: List<T> = ArrayList()
)